Frends.AmazonS3.DownloadObject/Frends.AmazonS3.DownloadObject/Definitions/Input.cs

```diff
@@ -0,0 +1,132 @@
+﻿using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+
+namespace Frends.AmazonS3.DownloadObject.Definitions
+{
+    /// <summary>
+    /// Connection and destination parameters.
+    /// </summary>
+    public class Input
+    {
+        /// <summary>
+        /// Authentication method to use when connecting to AWS S3 bucket. Options are pre-signed URL or AWS Access Key ID+AWS Secret Access Key.
+        /// </summary>
+        /// <example>AWSCredentials</example>
+        [DefaultValue(AuthenticationMethod.AWSCredentials)]
+        public AuthenticationMethod AuthenticationMethod { get; set; }
+
+        #region Pre-signed URL
+        /// <summary>
+        /// A pre-signed URL allows you to grant temporary access to users who don't have permission to directly run AWS operations in your account.
+        /// </summary>
+        /// <example>"https://bucket.s3.region.amazonaws.com/object/file.txt?X...</example>
+        [PasswordPropertyText]
+        [UIHint(nameof(AuthenticationMethod), "", AuthenticationMethod.PreSignedURL)]
+        [DefaultValue(null)]
+        public string PreSignedURL { get; set; }
+        #endregion Pre-signed URL
+
+        #region AWSCredentials
```

Comment:
No need for region here.

Reply:
This makes code easier to understand especially now that Options and Input are merged into Connection.